membership-common/src/main/scala/com/gu/memsub/subsv2/Subscription.scala

```diff
@@ -126,12 +126,14 @@ object GetCurrentPlans {
         case _ => ensureStarted(_.start <= dateToCheck)
       }
       val freePlanCancelled = alreadyStarted.ensure(DiscardedPlan(plan, "has a free plan which has been cancelled").wrapNel)(_)
-      val contributorPlanCancelled = alreadyStarted.ensure(DiscardedPlan(plan, "has a contributor plan which has been cancelled or removed").wrapNel)(_)
+      val contributorPlanCancelled =
```

Comment:
these changes must have been scalafmt.  I don't know if membership-common tests are run of `scalafmtCheck`'ed as membership-commmon is not aggregaed (although they do all pass so they might be)